### ISSUE TITLE
3.28.2-Firenze georeferencing not working as expected

### ISSUE THREAD

**Issue by CoosForest** [qgis/QGIS#51396]
### What is the bug or the crash?

I downloaded a 1958 single frame image from earthexplorer.usgs.gov. I used georeferencer and then opened the image in an existing qgis (3.28.2-Firenze) project. The referenced image appears about 9 miles south of where it should be. Longitude is quite close but not exact. Scale is correct.
I repeated this workflow several times to rule out operator error. The result is consistently off.
To complicate things, I cut cropped the image, so I had just the area I need. Georeferencer worked correctly on that image.
Seeing that I tried the full image again and got the wrong position again.
Suspecting that it is a bug, I georeferenced the larger image using 3.22.13-Białowieża and it worked perfectly.
Opened the project with newly referenced image in 3.28.2-Firenze and it appears correctly

### Steps to reproduce the issue

See above

### Versions

QGIS version
3.28.2-Firenze
QGIS code revision
b47e00ba601
Qt version
5.15.3
Python version
3.9.5
GDAL/OGR version
3.6.1
PROJ version
9.1.1
EPSG Registry database version
v10.076 (2022-08-31)
GEOS version
3.11.1-CAPI-1.17.1
SQLite version
3.39.4
PDAL version
2.4.3
PostgreSQL client version
unknown
SpatiaLite version
5.0.1
QWT version
6.1.6
QScintilla2 version
2.13.1
OS version
Windows 10 Version 2009

Active Python plugins
ImportPhotos
3.0.5
latlontools
3.6.7
QGISSortAndNumber-master
1.0
qgsAzimuth
0.9.15
shapetools
3.4.9
db_manager
0.1.20
MetaSearch
0.3.6
processing
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by agiudiceandrea** [comment]
@CoosForest, it seems to me the reported issue is the same as it was already previously reported: see https://github.com/qgis/QGIS/issues/51299, https://github.com/qgis/QGIS/issues/51197. Isn't it?

**Comment by CoosForest** [comment]
yes. #51299 <https://github.com/qgis/QGIS/issues/51299> is very similar

**Comment by agiudiceandrea** [comment]
Thanks for reporting. Closing as duplicate report.
